vnext/PropertySheets/External/Microsoft.ReactNative.Common.props

```diff
@@ -46,5 +46,7 @@
     <ResolveNuGetPackages>false</ResolveNuGetPackages>
   </PropertyGroup>
 
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\ReactTag.props" />
```

Comment:
I think you might be missing to add that file?
also, maybe we want to name it something more general if we're going to accumulate properties for API detection?